### PR TITLE
Remove fluentd_master and fluentd_node roles

### PR DESCRIPTION
**Pull request by abutcher**
Remove `fluentd_master` and `fluentd_node` roles as they are not being used and will be replaced with aggregated logging configuration.

### CODE REVIEW

File: roles/fluentd_master/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+embedded_etcd: "{{ openshift.master.embedded_etcd }}"
```
Comment (detiber) [inline roles/fluentd_master/vars/main.yml]: I'm not sure the related checks are even needed anymore with the test in the openshift_master role to verify that the api server is up before continuing.

As an aside, we should update for the proper package if it is available in the shipped repos as opposed to the public repo that it is currently being pulled from.

Comment (abutcher) [inline roles/fluentd_master/vars/main.yml]: Kk, I'll take the checks out.

**Comment by abutcher** [comment]
@twiest @wshearn is ops using the `fluentd_master`/`fluentd_node` roles via `openshift_use_fluentd=true`?

**Comment by twiest** [comment]
@abutcher I'll let @wshearn answer that.

**Comment by wshearn** [comment]
We are not using fluentd_{master,node}

**Comment by abutcher** [comment]
@detiber How do you feel about just making this work for now and dropping the roles when we add log aggregation bits

**Comment by detiber** [comment]
@abutcher truthfully, I'd rather yank this out if they aren't using it...  unless there is a valid case to do otherwise?

**Comment by abutcher** [comment]
@detiber Nah, I agree.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@detiber PTAL

**Comment by detiber** [comment]
@abutcher should probably kill the use_fluentd references in playbooks/{aws,gce,libvirt,openstack} as well, otherwise, :+1: from me

**Comment by abutcher** [comment]
@detiber Should be clear

```
<abutcher>@(strahd)[~/rhat/openshift-ansible] 16:10:26  (fluentd-fixes) 
$ grep -ir use_fluentd | wc -l
0
```

**Pull request merged**